torch/lib/THCS/generic/THCSTensor.c

```diff
@@ -275,6 +316,14 @@ THCSTensor *THCSTensor_(resize4d)(THCState *state, THCSTensor *self, long size0,
   return self;
 }
 
+void THCSTensor_(copy)(THCState *state, THCSTensor *self, THCSTensor *src) {
+  if (self == src) return;
+  THCSTensor_(rawResize)(state, self, src->nDimensionI, src->nDimensionV, src->size);
+  THCSTensor_(_set)(state, self, src->indices, src->values);
```

Comment:
That doesn't seem to be a good idea. Copying a tensor into another shouldn't make them share memory.